inclusive-design/wecount.inclusivedesign.ca
# fix: enhance inputs font size from 125 percent to 95 percent in uio button for mobile screens(resolves #336)

## Description
This PR fits all the button text on the UIO tab while UIO open dyslexic is turned on.


## Steps to test

1. Go to deploy preview
2. Inspect using iPhone 5 screen and Galaxy Fold screen
3. Turn on UIO open dyslexic and observe text on UIO tab
4. Turn on UIO enhanced inputs and observe text on UIO tab (UIO open dyslexic should remain on)

_make sure to do this for both the iPhone 5 screen and the Galaxy Fold screen_

**Expected behavior:** 
Button text should fit on UIO tab without overflowing and without being cut off.
## Additional information
N/A


## Related issues


closes #336
